src/Uno.UI.Composition/Composition/SingleFrameProvider.skia.cs

```diff
@@ -14,14 +14,18 @@ internal sealed class SingleFrameProvider : IFrameProvider
 
 	public SingleFrameProvider(SKImage image)
 	{
+		GC.AddMemoryPressure(image.Info.BytesSize);
```

Comment:
Let's link the uno issue here so we don't forget about it (we don't want to double the pressure)